monitor/monitor.go

```diff
@@ -112,7 +112,7 @@ func (m *Monitor) handleConnection(server net.Listener) {
 
 		mutex.Lock()
 		listeners.PushBack(conn)
-		log.Infof("New monitor connected. len(listeners) = %d", listeners.Len())
+		log.WithField("count.listener", listeners.Len()).Info("New monitor connected.")
```

Comment:
Remove the period at the end of the log message. It's not consistent with the other messages.